Different error:
```
numpy\typing\tests\test_runtime.py:89: in <module>
    "_SupportsArray": (_npt._SupportsArray, np.arange(10)),
                                            ^^^^^^^^^^^^^
E   ValueError: arange: cannot compute length
        Any        = typing.Any
        NamedTuple = <function NamedTuple at 0x000001CA11391300>
        Self       = typing.Self
        TYPES      = {'ArrayLike': TypeTup(typ=collections.abc.Buffer | _DualArrayLike[numpy.dtype, complex | bytes | str], args=(<class 'c...umpy.ndarray[_AnyShape, numpy.dtype[ScalarT]], args=(_AnyShape, numpy.dtype[ScalarT]), origin=<class 'numpy.ndarray'>)}
```
Not sure what that is about.

--- a/.github/workflows/typecheck.yml
+++ b/.github/workflows/typecheck.yml
@@ -22,7 +22,7 @@ on:
       - '**.rst'
       - '.circlecl/**'
       - '.devcontainer/**'
-      - '.spin/**'
+      - '.spin/LICENSE'
       - 'benchmarks/**'
       - 'branding/**'
       - 'doc/**'
@@ -54,7 +54,7 @@ jobs:
         os_python:
           - [macos-latest, '3.14']
           - [ubuntu-latest, '3.13']
-          - [windows-latest, '3.12']
+          - [windows-2022, '3.12']
     steps:
     - uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10 # v6.0.3
       with:

